### DISCUSSION TITLE
cara pasang script di hosting / cpanel buat kirim OTP

### DISCUSSION THREAD

**Discussion by azaingo** [kirimwa-id/forum#70] [Umum]
bismillah, maap sy pemula utk ini, bagaimana cara memasang sript nya buat kirim OTP whatsapp ya?
sy baru upload di hosting pakai fw CI3, critanya pelanggan mau beli produk nt diarahkan ke halaman daftar akun baru yaitu mengisi data user baru yg mana nanti dpt OTP email sama whatsapp, utk email sudah bisa verifikasi pakai cronjobs..., nah kurang satu lagi apakah bisa dengan menerapkan yg ada di kirim wa bisa sperti itu?

atas jawabannya saya ucapkan  banyak terimakasih & jazakumullahukhair

**Comment by fahram** [comment]
di dokumentasinya sudah ada contoh source code php.
[https://developer.kirimwa.id/api/v1/#post-messages](https://developer.kirimwa.id/api/v1/#post-messages)

**Reply by azaingo** [reply]
bismillah, mas fahram cara untuk meletakkan filenya dimana ya naruhnya..., ada tutorialnya...?

**Reply by fahram** [reply]
tidak harus buat file sendiri, bisa ditaruh di file yang sama dengan logic kirim emailnya.
saya rasa dokumentasinya sudah cukup jelas sih.

**Comment by rioastamal** [comment]
Coba lihat diskusi #68 untuk contoh

**Reply by azaingo** [reply]
ya mas... ini sy coba dulu mas,

**Reply by azaingo** [reply]
hasilnya ini mas... setelah tak coba dilocalhost
404
Mohon maaf, halaman yang Anda cari tidak ditemukan. Silahkan kembali ke halaman sebelumnya atau ke beranda

**Reply by azaingo** [reply]
yg dimaksud pinginnya dari web toko online bisa kirim kode token aja buat verifikasi login baru, saat ini yg bisa masuk email aja, klw model laen berbayar mas kaya wablas dkk

**Reply by azaingo** [reply]
kode token whatsapp, yang mana tadi sudah daftar buat akun baru trus ada verifikasi no whatsapp

**Reply by azaingo** [reply]
saya sudah dapat kiriman kode token dari Token API KirimWA.id, makainya gimana ya mas

**Reply by rioastamal** [reply]
Pengguna API ini harus paham bagaimana melakukan HTTP request dengan PHP dan format JSON secara umum. Jika belum paham maka silahkan pelajari basic itu dulu atau bisa minta orang lain untuk melakukan integrasi dengan API KirimWA.id

**Comment by fahram** [comment]
> yg dimaksud pinginnya dari web toko online bisa kirim kode token aja buat verifikasi login baru, saat ini yg bisa masuk email aja, klw model laen berbayar mas kaya wablas dkk

Masnya buat module di webnya mas. Tinggal request ke server api.kirimwa.id bisa pakai curl atau http library seperti guzzle.

**Reply by azaingo** [reply]
waduh.. mas ada inbox wa ? masih gak paham itu mas, mgkn bisa dikasih contoh mas caranya...

**Comment by azaingo** [comment]
ya mas, makasih banyak masukannya mas..., maklum masih copas2...
